tests/S3/SSECMiddlewareTest.php

```diff
@@ -71,6 +71,20 @@ public function getListenerTestCases()
                     'SSECustomerKeyMD5' => null,
                 ]
             ],
+            [
+                'PutObject',
```

Comment:
This update needs to supply similar behavior for `ListObjects` and `CopyObject` as the tests above it.